model/utils.py

```diff
@@ -0,0 +1,28 @@
+from functools import wraps
+import json
+import time
+import os
+
+from model.logger import logger
+
+def timing(f):
+    @wraps(f)
+    def wrap(*args, **kw):
+        ts = time.time()
+        result = f(*args, **kw)
+        te = time.time()
+        logger.info('func:%r args:[%r, %r] took: %2.4f sec' % \
+          (f.__name__, args, kw, te-ts))
+        return result
+    return wrap
+
+def map_reactions_list(map_path):
```

Comment:
I'm not sure it belongs to `utils`, maybe not general enough?

Reply:
Good point, I moved this to response